cms/static/cms/js/modules/cms.structureboard.js

```diff
@@ -59,6 +59,7 @@ class StructureBoard {
         this.state = false;
         this.dragging = false;
         this.latestAction = [];
+        this.trigger_window_load_event = true;
```

Comment:
this doesn't seem to be used

Reply:
Must have been in there at some point and I forgot to remove it. After nearly half a year this looks like an artifact to me as well. I will remove it and see what happens.

Reply:
You were right, it was an artifact.